MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.ComboBox.xaml

```diff
@@ -365,10 +365,10 @@
                                 <Condition Binding="{Binding IsMouseOver, RelativeSource={RelativeSource Self}}" Value="true" />
                                 <Condition Binding="{Binding IsEditable, RelativeSource={RelativeSource AncestorType={x:Type ComboBox}}}" Value="true" />
                             </MultiDataTrigger.Conditions>
-                            <Setter Property="BorderBrush" Value="{DynamicResource PrimaryHueMidBrush}" />
+                            <Setter Property="BorderBrush" Value="{Binding Path=(wpf:TextFieldAssist.UnderlineHighlightedBrush), RelativeSource={RelativeSource AncestorType={x:Type ComboBox}}}" />
                         </MultiDataTrigger>
                         <Trigger Property="IsPressed" Value="true">
-                            <Setter TargetName="arrow" Property="Fill" Value="{DynamicResource PrimaryHueDarkBrush}" />
+                            <Setter TargetName="arrow" Property="Fill" Value="{Binding Path=(wpf:TextFieldAssist.UnderlineHighlightedDarkBrush), RelativeSource={RelativeSource AncestorType={x:Type ComboBox}}}" />
```

Comment:
This is just changing the color of the drop down arrow. I don't think we want to use the `UnderlineHighlightedDarkBrush` brush here and just leave it as the `{DynamicResource PrimaryHueDarkBrush}`